File: 241022/시력 검사 2/eye-test-2.cpp
#include <iostream>
using namespace std;

int main() {
    double eyesight;
    cin >> eyesight;

    if (eyesight >= 1.0) {
        cout << "High";
    } else if (eyesight >= 0.5) {
        cout << "Middle";
    } else {
        cout << "Low";
    }
    return 0;
}

File: 241022/시력 검사 2/eye-test-2.js
const fs = require("fs");
let input = fs.readFileSync(0).toString().trim();
let eyesight = Number(input);

if (eyesight >= 1.0) {
    console.log("High");
} else if (eyesight >= 0.5) {
    console.log("Middle");
} else {
    console.log("Low");
}